src/Resources/contao/library/Contao/Widget.php

```diff
@@ -381,11 +381,11 @@ public function __get($strKey)
 
 			case 'value':
 				// Encrypt the value
-				if ($this->arrConfiguration['encrypt'])
+				if (isset($this->arrConfiguration['encrypt']) && true === $this->arrConfiguration['encrypt'])
 				{
 					return \Encryption::encrypt($this->varValue);
 				}
-				elseif ($this->varValue == '')
+				elseif ($this->varValue === '')
```

Comment:
It is. `$this->varValue == '')` would be `true` if `$this->varValue` is `null` ;-) A case I would have never thought of without unit tests ;-)